Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -58,7 +58,7 @@ namespace
 	const FString StatProfileCommandLineKey = TEXT("-StatProfile=");
 
 	const FString MemReportFlag = TEXT("mem_report");
-	const FString MemRemportIntervalKey = TEXT("-Interval=");
+	const FString MemRemportIntervalKey = TEXT("-MemReportInterval=");
```

Comment:
What is this -MemReportInterval?

Reply:
It's a key used in command arguments (eg.-MemReportInterval=30). I renamed this key to make it less confusing.